rules/palettes.yaml

```diff
@@ -20,6 +20,30 @@
 	PaletteFromCurrentTileset@decoration:
 		Name: terraindecoration
 		ShadowIndex: 1
+	PaletteFromFile@apolblue:
+		Name: apolblue
+		Filename: apolblue.pal
+		Alpha: 0.8
```

Comment:
Also applies to existing `PaletteFromFile@alpha2:`. Needs feedback from @trempler.